Module1/todo_app/trello.py

```diff
@@ -2,19 +2,22 @@
 from dotenv import load_dotenv
```

Comment:
Similar to a previous comment - I think this was for testing (which is a really sensible approach!) but we can now delete this in favour of keeping the actual codebase the source of truth?